app/controllers/releases_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ReleasesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+  before_action :set_release, only: [:edit, :update, :show, :destroy]
+
```

Comment:
You need to check to see if the current user is an admin before you allow them to perform these actions

`before_action :is_admin?`

Reply:
<img width="1280" alt="Screen Shot 2022-02-02 at 1 49 11 PM" src="https://user-images.githubusercontent.com/81119399/152243147-8ee7715f-4d6a-425f-9bb3-50a8e3ca2c2a.png">